test/readMPAS.f90

```diff
@@ -30,6 +30,137 @@ function epsilonDiff(a,b) result(isSame)
   endif
 end function
 
+
+subroutine rebuildTests(mpMesh, numMPs, mp2Elm, isMPActive)
+    use :: polympo
+    use iso_c_binding
+    implicit none
+    type(c_ptr):: mpMesh
+    integer :: numMPs, i, numMPsLarger
+    integer, dimension(:), pointer :: mp2Elm, addedMPMask, isMPActive, mp2ElmFromPMPO
+    integer, dimension(:), pointer :: mp2ElmLarger, addedMPMaskLarger, mp2ElmFromPMPOLarger, isMPActiveLarger
+    integer, parameter :: MP_ACTIVE = 1
+    integer, parameter :: MP_INACTIVE = 0
+    integer, parameter :: MP_DELETE_ELM_ID = -1
+
+    ! TEST: adding MP
+    ! Necessary pre-conditions for test
+    call assert(numMPs >= 4, "not enough MPs for test")
+    call assert(isMPActive(4) == MP_INACTIVE, "mp2Elm = 4 is active")
+    ! PREPARE DATA
+    allocate(addedMPMask(numMPs))
+    addedMPMask = MP_INACTIVE
+    isMPActive(4) = MP_ACTIVE
```

Comment:
How does the assertion on line 49 matter if we change the settings for this MP anyway?  The same question applies to the other test cases below.

Reply:
See comment above